secretsstoresynccontroller/config/admissionpolicies/update_secret_sync_object_exists_for_kubernete_secret.yaml

```diff
@@ -0,0 +1,34 @@
+# This policy is used to prevent updating a secret without one and only one secret owner,
+# which has the type SecretSync.
+# TODO: Use parameters to allow the user to specify the name of the SecretSyncController.
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "secretsynccontroller-update-ownership-policy"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   [""]
+      apiVersions: ["v1"]
+      operations:  ["UPDATE"]
+      resources:   ["secrets"]
+  variables:
+  - name: hasOneSecretSyncOwner
+    expression: "((size(oldObject.metadata.ownerReferences) == 1) && (size(object.metadata.ownerReferences) == 1))" 
+  - name: hasSameOwner
+    expression: "(object.metadata.ownerReferences.all(e, (e in oldObject.metadata.ownerReferences) && e.kind == 'SecretSync') == true)"
```

Comment:
I think we can collapse this to the create policy

- on create, make sure the controller only sets a single ownerReference to the SecretSync object
- on update, make sure the controller is setting an ownerReference to the SecretSync object and is not adding other ownerReferences that weren't already there?